source/linux/Foundational_Components/U-Boot/UG-Network-K3.rst

```diff
@@ -50,6 +52,24 @@ binaries files to be sent over TFTP are listed in the table below.
         - Disable link info Bootmode pin so that ROM can identify the PHY and establishes link with the supported speed and duplex mode.
         - Please note that due to MDIO corruption (Errata i2329), booting over Ethernet is not recommended for production purposes.
 
+.. ifconfig:: CONFIG_part_variant in ('J722S')
+
+    +------------------+--------------------------------------+-----------------------+------------------------+---------------------------+----------------------------+
+    | Board            | Defconfigs                           | Supported interfaces  | ROM                    | R5 SPL                    | A53 SPL                    |
+    +==================+======================================+=======================+========================+===========================+============================+
+    | J722s EVM        | ``j722s_evm_r5_ethboot_defconfig``   | CPSW NUSS Ethernet    | VCI:  TI K3 Bootp Boot | VCI: J722S U-Boot R5 SPL  | VCI: J722S U-Boot A53 SPL  |
+    |                  | ``j722s_evm_a53_ethboot_defconfig``  |                       | :file:`tiboot3.bin`    | :file:`tispl.bin`         | :file:`u-boot.img`         |
+    +------------------+--------------------------------------+-----------------------+------------------------+---------------------------+----------------------------+
```

Comment:
This table should align with the directive and the below section. Remove a space.